### PR TITLE
Check for numeric strings

### PR DESCRIPTION
**Pull request by exussum12**
String which are simply numbers should be treated as numbers

eg '1234' should be treated like 1234.

other non numeric strings are still treated as they were

### CODE REVIEW

File: tests/DetectorTest.php [hunk 1]
```diff
@@ -70,10 +70,11 @@ public function testDetectWithAssignExtension()
         $detector = $this->createDetector($option);
         $fileReport = $detector->detect(FileReportTest::getTestFile('test_1'));
 
+        var_dump($fileReport->getEntries());
```
Comment (localheinz) [inline tests/DetectorTest.php]: ?

Comment (exussum12) [reply tests/DetectorTest.php]: Sorry left debug in, its been removed

**Comment by povils** [comment]
First of all thanks. Secondly, I'm thinking maybe pass the option "--include-numeric-string" instead of breaking peoples CIs ? :) Also could you rebase your commits.

**Comment by exussum12** [comment]
Commit rebased, its also an option on merging btw to do a rebase merge.
I will look at making it an option later on today.

**Comment by exussum12** [comment]
Made a command but had no idea why that test is failing, The test seems flaky to depend on the number 11 ?

**Comment by exussum12** [comment]
Fixed the test

**Comment by povils** [comment]
Big thanks!!

**Pull request merged**